controller/metrics/prommetrics.go

```diff
@@ -111,14 +111,14 @@ var (
 	MetricAnalysisRunMetricType = prometheus.NewDesc(
 		"analysis_run_metric_type",
 		"Information on the type of a specific metric in the Analysis Runs",
-		append(namespaceNameLabels, "metric", "type"),
+		append(namespaceNameLabels, "metric", "type", "dryRun"),
```

Comment:
TBH, i think `MetricAnalysisRunMetricType` is redundant to `MetricAnalysisRunMetricPhase` so we can stop adding to this metric and consider deprecating in the future. So you can remove the dry run label here.

Reply:
Removed.